array-methods-basic/basic-chaining.js

```diff
@@ -4,13 +4,21 @@
 var vowels = ["a", "e", "i", "o", "u"];
 
 function getUpperCaseVowels(letters) {
-    var result = [];
-    for(var i = 0; i < letters.length; i++) {
-        if(vowels.indexOf(letters[i]) !== -1) {
-            result.push(letters[i].toUpperCase());
-        }
-    }
-    return result;
+    const getVowel = letters.filter(function (vowel) {
```

Comment:
The parameter in the callback function should be named `letter` as the array has not yet been filtered.